bootstrap-app/src/cmr/bootstrap/system.clj

```diff
@@ -28,7 +27,9 @@
    [cmr.metadata-db.config :as mdb-config]
    [cmr.metadata-db.system :as mdb-system]
    [cmr.oracle.connection :as oracle]
-   [cmr.transmit.config :as transmit-config]))
+   [cmr.transmit.config :as transmit-config]
+   [cmr.bootstrap.services.dispatch.core-async-dispatch :as core-async-dispatch]
+   [cmr.bootstrap.services.dispatch.synchronous-dispatch :as synchronous-dispatch]))
```

Comment:
With the changes mentioned above, you wouldn't need to import both of these; instead, you'd just do:
```
  [cmr.bootstrap.services.dispatch.core :as dispatch]
```
and then below you'd call `(dispatch/<some factory function> config-data)` and then create the appropriate dispatcher given the config values.